### PR TITLE
Recent donation for DEN

### PR DESCRIPTION
**Pull request by DenDev-247**
most recent donation for DEN pool

### CODE REVIEW

File: DEN/donations.yml [hunk 1]
```diff
@@ -51,3 +51,29 @@
...
+    tel: "+62 (08) 6456 2222"
+    contact: info@pchf.org.au
+  ada: 34
+  usd: 41.85
+  timestamp: 1627210800
+  files:
+    - DEN/pchf5.pdf
+  announcement_url: https://denadapool.io/donations.html
+  
+  - ticker: DEN
```
Comment (nova4005) [inline DEN/donations.yml:68]: I think this is the only error in the YAML file, but just double check it with [YAML](https://codebeautify.org/yaml-validator/) validator to be sure.

You may also need to update your repo with the main branch to make sure it is not out of date. Instructions are in the [README.md](https://github.com/CardanoMDP/Donations#keeping-your-repository-up-to-date-with-the-upstream-repository-cardanomdpdonations)

**Comment by DenDev-247** [comment]
Hi There, I've updated the pull request. Is there anything else that needs to be done?

Regards ;)

**Review by aloopmarin007 — APPROVED** [review]
LGT

**Comment by aloopmarin007** [comment]
Hi, you have some some conflicts in donations.yml which must be solved

**Comment by DenDev-247** [comment]
Sorry Marin, can you elaborate?

**Comment by aloopmarin007** [comment]
Hi Al, your donations.yml is conflicting with main and we cannot merge it. You need to find the error in your donations.yml and fix it.

**Pull request closed without merging**